filter/rastertoepson.c

```diff
@@ -877,7 +874,7 @@ OutputRows(
 
     n = dot_count / DotBytes;
     putchar((int)(n & 255));
-    putchar((int)(n / 256));
+    putchar((int)(n >> 8));
```

Comment:
Cosmetic since a good compiler should convert this to a shift anyways.

Reply:
Yes but you use bitwise for the &255, so it should be clear that you putchar the low bits and then putchar the high bits